security/policykit/Portfile

```
@@ -5,11 +5,8 @@ PortGroup           gobject_introspection 1.0
 PortGroup           compiler_blacklist_versions 1.0
 PortGroup           legacysupport 1.0
 
-# pin version at 0.116 for now
-# upstream version 0.117 is an unstable prerelease and is not recommended
-# see https://gitlab.freedesktop.org/polkit/polkit/-/commit/585f4f2715639394e36319d4918389d26e250e7b
 name                policykit
-version             0.116
+version             0.120
 revision            1
```

When the version is increased, the revision should be reset to 0.
```suggestion
revision            0
```